packages/astro/src/@types/astro.ts

```diff
@@ -2166,6 +2166,37 @@ export interface AstroUserConfig {
 			 * ```
 			 */
 			schema?: EnvSchema;
+
+			/**
+			 * @docs
+			 * @name experimental.env.validateSecrets
+			 * @kind h4
+			 * @type {boolean}
+			 * @default `false`
+			 * @version 4.11.6
```

Comment:
just pointing out this looks right to me if this is merged now, but if it misses the next patch, you'll need to update the version number here!